examples/cifar10/train.cpp

```diff
@@ -26,16 +26,20 @@ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
+#include <cstdlib>
 #include <iostream>
 #include <vector>
 #include "tiny_dnn/tiny_dnn.h"
 
+#ifdef _MSC_VER
+#define strcasecmp _stricmp
+#endif
+
 using namespace tiny_dnn;
 using namespace tiny_dnn::activation;
-using namespace std;
 
 template <typename N>
-void construct_net(N &nn) {
+void construct_net(N &nn, core::backend_t backend_type) {
```

Comment:
It probably can't, but I'm not sure. Once again, it was here before, I believe @nyanp is the author of it.